server/scsynth/SC_Graph.cpp

```diff
@@ -431,15 +431,28 @@ void Graph_Ctor(World *inWorld, GraphDef *inGraphDef, Graph *graph, sc_msg_iter
 	inGraphDef->mRefCount ++ ;
 }
 
+// Calls the unit's Ctor func.
+inline void InitializeUnit(Unit* unit)
+{
+	try {
+		(*unit->mUnitDef->mUnitCtorFunc)(unit);
```

Comment:
In this case, isn't the exception being both thrown and caught inside of libscsynth? There isn't another shared library involved, is there? Though - `catch (std::exception)` does open the door to accidentally catching something thrown by the plugin.